From f207badcf48dc620c32f406783b13603a496ce4d Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Tue, 22 Oct 2024 22:39:04 +0300
Subject: [PATCH 14/19] Revert "base: Don't use updatable apexes"

Starting from Android-15 updatable apexes are mandatory.

This reverts commit 82691324c2c4199a51e866815e18ad223aada7b2.
---
 common/base/device.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/common/base/device.mk b/common/base/device.mk
index 2e78d23..45f3137 100644
--- a/common/base/device.mk
+++ b/common/base/device.mk
@@ -18,6 +18,9 @@ ifneq ($(filter $(TARGET_ARCH),arm64 x86_64),)
     $(call inherit-product, $(SRC_TARGET_DIR)/product/core_64_bit.mk)
 endif
 
+# Enable updating of APEXes
+$(call inherit-product, $(SRC_TARGET_DIR)/product/updatable_apex.mk)
+
 # Enable userspace reboot
 $(call inherit-product, $(SRC_TARGET_DIR)/product/userspace_reboot.mk)
 
-- 
2.43.0

